Lets Do This/Networking/DSOAPI.m

```diff
@@ -174,27 +174,30 @@ - (void)logoutWithCompletionHandler:(void(^)(NSDictionary *))completionHandler e
     }];
 }
 
-- (void)createCampaignSignupForCampaign:(DSOCampaign *)campaign completionHandler:(void(^)(DSOCampaignSignup *))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
-    NSString *url = [NSString stringWithFormat:@"user/campaigns/%ld/signup", (long)campaign.campaignID];
-    NSDictionary *params = @{@"source": LDTSOURCENAME};
-
+- (void)postSignupForCampaign:(DSOCampaign *)campaign completionHandler:(void(^)(DSOCampaignSignup *))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
+    NSDictionary *params = @{@"campaign_id" : [NSNumber numberWithInteger:campaign.campaignID], @"source" : LDTSOURCENAME};
+    NSString *url = @"signups";
     [self POST:url parameters:params success:^(NSURLSessionDataTask *task, id responseObject) {
-        DSOCampaignSignup *signup = [[DSOCampaignSignup alloc] initWithDict:responseObject[@"data"]];
-        signup.campaign = campaign;
+        NSLog(@"responseObject %@", responseObject);
+        // @todo This is my insane API endpoint we should change to return an actual Signup object
+        // This seems to fial, cant figure out how to get responseObject in value with ( parenthesis )
+//        NSInteger signupID = (NSInteger)responseObject;
+        DSOCampaignSignup *signup = [[DSOCampaignSignup alloc] init];
         if (completionHandler) {
             completionHandler(signup);
         }
-      } failure:^(NSURLSessionDataTask *task, NSError *error) {
-          [self logError:error methodName:NSStringFromSelector(_cmd) URLString:url];
-          if (errorHandler) {
-              errorHandler(error);
-          }
-      }];
+    } failure:^(NSURLSessionDataTask *task, NSError *error) {
+        [self logError:error methodName:NSStringFromSelector(_cmd) URLString:url];
+        if (errorHandler) {
+            errorHandler(error);
+        }
+    }];
 }
 
 - (void)postReportbackItem:(DSOReportbackItem *)reportbackItem completionHandler:(void(^)(NSDictionary *))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
-    NSString *url = [NSString stringWithFormat:@"user/campaigns/%ld/reportback", (long)reportbackItem.campaign.campaignID];
+    NSString *url = @"reportbacks";
     NSDictionary *params = @{
+                             @"campaign_id": [NSNumber numberWithInteger:reportbackItem.campaign.campaignID],
```

Comment:
@DFurnes This results in a `Request failed: internal server error (500)` for any campaign I try to reportback for on staging (refs https://github.com/DoSomething/northstar/pull/273)